package.json

```
@@ -15,8 +15,8 @@
   "author": "",
   "license": "ISC",
   "engines": {
-    "node": ">=14 <20",
-    "npm": ">=7.20.x <9"
+    "node": ">=18 <20",
+    "npm": ">=9 <10"
```

I believe node 16 uses npm 8

```suggestion
    "npm": ">=8 <10"
```